expenses-manager-frontend/src/main/frontend/src/app/expense-table/expense-table.component.scss

```diff
@@ -0,0 +1,128 @@
+.expense-table{
```

Comment:
Din ce vad in Figma tabelul, bara de data si paginarea au cate un border si un shadow in spate, il putem adauga te rog?